DependencyInjection/MonologExtension.php

```diff
@@ -14,6 +14,7 @@
 use Monolog\Attribute\AsMonologProcessor;
 use Monolog\Handler\FingersCrossed\ErrorLevelActivationStrategy;
 use Monolog\Handler\HandlerInterface;
+use Monolog\Level;
```

Comment:
I tried an other implementation with the enum `Monolog\Level::fromName`, but that does not work if the env var or parameter contains a number.

https://github.com/symfony/monolog-bundle/pull/446/commits/2f92e7400910a4344044d7baf98dbb952d98eb17